### ISSUE TITLE
Move the containers contrib to coq-community

### ISSUE THREAD

**Issue by Zimmi48** [coq-community/manifesto#45]
## Move a project to coq-community ##

**Project name:** Containers

**Initial author(s):** Stéphane Lescuyer @StekiKun

**Current URL:** https://github.com/coq-contribs/containers

**Kind:** pure Coq library

**License:** TBD by the author before the transfer

**Description:** a typeclass-based library of finite sets/maps

**Status:** was recently maintained by various contributors as part of coq-contribs

**New maintainer:** the original author, @StekiKun, will be the principal maintainer for the time being

Other interested folks include: @siddharthist, @sigurdschneider, @vbgl

**Comment by vbgl** [comment]
About the *kind* line. This project also includes a plugin that is supposed to automatically derive `OrderedType` instances from user-provided type definitions.

**Comment by StekiKun** [comment]
Indeed, and this is the part that is harder to maintain as the Coq API or Ltac behaviour evolve. I opened coq-contribs/containers#8 to make sure we start with something that works before moving to coq-community.

**Comment by Zimmi48** [comment]
Oops sorry, I filled this out too fast.

**Comment by Zimmi48** [comment]
FTR I can merge the PR if needed but I don't feel competent to review it. Maybe @vbgl could do it too?

**Comment by siddharthist** [comment]
I'm happy to help out with the maintenance and development of the included proof scripts, though I'm not very knowledgeable about plugins.

**Comment by Zimmi48** [comment]
So shall I just go ahead and merge coq-contribs/containers#8?

**Comment by Zimmi48** [comment]
@StekiKun I'm still waiting on your answers. I can merge coq-contribs/containers#8 (without review) or I can let you merge it after transferring the repository.

However, before transferring the repository, I first need you to add an open source license of your choosing to it. It can be any license, but if you have no idea which license to choose, I'd suggest a permissive license like MIT.

**Comment by vbgl** [comment]
Is there any hope to see progress on this issue? IMHO it would be sad to loose this library…

**Comment by Zimmi48** [comment]
The only thing blocking the transfer is the lack of a license (https://github.com/coq-contribs/containers/issues/4). @StekiKun has not responded to the GitHub notifications about this so maybe @vsiles can help us transfer the request again.

**Comment by vsiles** [comment]
Will ask again !
